datarepo-clienttests/src/main/java/runner/config/ServerSpecification.java

```diff
@@ -27,6 +27,7 @@
   public ServiceAccountSpecification testRunnerServiceAccount;
   public boolean skipKubernetes = false;
   public boolean skipDeployment = false;
+  public boolean dbDropAllOnStart = true;
```

Comment:
Will this ever get run against prod, even accidentally?

Reply:
Great point - I've switched it to default to "false"